lib/pbench/server/api/resources/__init__.py

```diff
@@ -755,6 +756,28 @@ class ApiMethod(Enum):
     PUT = auto()
 
 
+class ApiAttributes:
```

Comment:
As a side note, here, this is a great use for the `@dataclass` decorator. Instead of writing an `__init__` constructor that explicitly copies parameters to identically named attributes, you just define the attributes and the dataclass infrastructure does the rest:

```
@dataclass
class ApiAttributes:
    action: str
    operation_name: OperationName
    require_stable: bool
    require_map: bool
```